packages/design-system/src/styles/components/_Choice.scss

```diff
@@ -165,6 +173,23 @@ label,
 
   &:checked {
     border-color: $choice__background-color--checked;
+
+    &:disabled {
+      border-color: $choice__border-color--disabled;
+
+      &::before {
+        box-shadow: inset 0 0 4em 1em $choice__border-color--disabled;
+      }
+
+      @media (-ms-high-contrast: active), (forced-colors: active) {
+        border-color: GrayText;
+
+        &::before {
+          background-color: GrayText;
+          box-shadow: inset 0 0 4em 1em GrayText;
```

Comment:
`box-shadow` isn't respected by HCM, so I don't think this is doing anything here.

Reply:
Good call!